streams/src/main/java/org/apache/kafka/streams/kstream/KGroupedStream.java

```diff
@@ -578,7 +578,7 @@
      * (rolling) aggregate for each key
      */
     <VR> KTable<K, VR> aggregate(final Initializer<VR> initializer,
-                                 final Aggregator<K, V, VR> aggregator,
+                                 final Aggregator<? super K, ? super V, VR> aggregator,
```

Comment:
A meta clarification question: why we do not extend the return value type in `aggregate` and `groupBy`?